src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -155,6 +157,25 @@ public function get_string_name( $node_id, $field, $settings ) {
 		return $field['field'] . '-' . $settings[ $this->type ] . '-' . $node_id;
 	}
 
+	/**
+	 * Get wrap tag for string.
+	 * Used for SEO, can contain (h1...h6, etc.)
+	 *
+	 * @param array $settings Field settings.
+	 *
+	 * @return string
+	 */
+	private function get_wrap_tag( $settings ) {
+		if ( isset( $settings[ $this->type ] ) && 'heading' === $settings[ $this->type ] ) {
+			// h2 must be by default, as Elementor sends empty header size for h2 block.
```

Comment:
@kagg-design You could replace this comment with a named constant.

```
const DEFAULT_HEADING_TAG = 'h2';

...
...
$header_size = isset( $settings[ $this->settings_field ]['header_size'] ) ? $settings[ $this->settings_field ]['header_size'] : self::DEFAULT_HEADING_TAG;
```

Reply:
@brucepearson done